docs/getting-started-guides/vsphere.md

```diff
@@ -79,8 +79,7 @@ Sample Config:
 
 #### Known issues
 
-* [Volumes are not removed from a VM configuration if the VM is down](https://github.com/kubernetes/kubernetes/issues/33061). The workaround is to manually remove the disk from VM settings before powering it up.
-* [FS groups are not supported in 1.4.7](https://github.com/kubernetes/kubernetes/issues/34039) - This issue is fixed in 1.4.8
+* [Unable to execute command on pod container using kubectl exec] (https://github.com/kubernetes/kubernetes-anywhere/issues/337)
```

Comment:
Typo causing link not to be rendered correctly: remove space between ']' and '('